modular_skyrat/modules/customization/modules/clothing/under/security.dm

```diff
@@ -97,7 +52,38 @@
 	mutant_variants = NONE
 	can_adjust = FALSE
 
-/// SEC GENERAL
+///CDO
```

Comment:
CDO doesn't exist, just axe this

Reply:
Its there for the police uniforms that once belonged to CDO and that I'm not removing bc its used for other stuff, but yeah I'll change the comment at the least